library/src/blas3/rocblas_trmm.hpp

```diff
@@ -0,0 +1,1488 @@
+/* ************************************************************************
+ * Copyright 2019 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+#include "../blas1/rocblas_copy.hpp"
+#include "../blas1/rocblas_scal.hpp"
+#include "../blas2/rocblas_gemv.hpp"
+#include "../blas3/Tensile/gemm.hpp"
+#include "dcld.hpp"
+#include "handle.h"
+#include "rocblas.h"
+#include "utility.h"
+
+namespace
+{
```

Comment:
Nnnamed namespaces should not appear in header files.

Reply:
Removed unnamed namespace.